src/main/java/com/crowdin/cli/utils/Utils.java

```diff
@@ -87,7 +87,12 @@ public static String buildUserAgent() {
     }
 
     public static String unixPath(String path) {
```

Comment:
This method is only meant to be used on Unix platforms, isn't it? 

We have a separate `windowsPath` method below.

Something is wrong with the `isWindows` condition inside the `unixPath` method.